pages/common/helm.md

```diff
@@ -0,0 +1,39 @@
+# helm
+
+> Helm is a package manager for Kubernetes, that helps in defining, installing and updating k8s applications.
```

Comment:
We can move the second part of the description as a sub-heading. It is too long for a single line.